contracts/src/CrossChainApplications/examples/NativeTokenBridge/NativeTokenDestination.sol

```diff
@@ -41,12 +40,10 @@ contract NativeTokenDestination is TeleporterOwnerUpgradeable, INativeTokenDesti
     address public constant BURNED_TX_FEES_ADDRESS = 0x0100000000000000000000000000000000000000;
 
     /**
-     * @notice Designated Blackhole Address for this contract.
-     *
-     * @dev Tokens are sent here to be "burned" before sending an unlock message to the source chain. Different
-     * from the burned tx fee address so they can be tracked separately.
+     * @dev Used when reporting burned gas fees to the source chain. Fees will be unlocked and sent to this address.
+     * Different from the burned tx fee address so they can be tracked separately.
```

Comment:
```suggestion
     * @notice Address used by this contract to blackhole funds, effectively burning them.
     * @dev Native tokens are burned by this contract by sending them to this address when transferring tokens back to
     * the source chain. When reporting burned transaction fee amounts, this address is used as the recipient
     * address for the funds to be sent to be burned on the source chain.
     * Different from BURNED_TX_FEES_ADDRESS so that the total amount burned in transaction fees and the amount 
     * total amount burned to be sent back to the source chain can be tracked separately. 
```